hg-fast-export.py

```diff
@@ -122,9 +135,31 @@ def get_author(logmessage,committer,authors):
       return r
   return committer
 
+def generate_git_lfs_pointer(filename):
+  size = 0
+  hasher = hashlib.sha256()
+  with open(filename, "rb") as bfile:
+    buf = bfile.read(1024*1024*1024)
+    while buf:
+      hasher.update(buf)
+      size = size + len(buf)
+      buf = bfile.read(1024*1024*1024)
```

Comment:
It is true that 1Gib of memory is a pretty big chunk, I'll make sure to reduce this and create a constant for it.